components/collective-page/sections/Contribute.js

```diff
@@ -380,69 +380,71 @@ class SectionContribute extends React.PureComponent {
 
             {/* Events, for now (til v2 standalone section) */}
             {hasOtherWaysToContribute && !newNavbarFeatureFlag && (
-              <HorizontalScroller getScrollDistance={this.getContributeCardsScrollDistance}>
-                {(ref, Chevrons) => (
-                  <div>
-                    <ContainerSectionContent>
-                      <Flex justifyContent="space-between" alignItems="center" mb={3}>
-                        <H3 fontSize="20px" fontWeight="600" color="black.700">
-                          {connectedCollectives.length > 0 ? (
-                            <FormattedMessage
-                              id="SectionContribute.MoreWays"
-                              defaultMessage="More ways to contribute"
+              <Box pb={4}>
```

Comment:
We shouldn't worry about the old navbar, it's about to be removed and we won't rollback at this point.